### PR TITLE
Adds Default options in wp_options table to allow manipulating options from WP CLI.

### PR DESCRIPTION
**Pull request by Vedant-Gandhi**
## Description
The aim of this PR is to add support for purging AMP URL's.

## Type
- New Feature

## Reference Issue
- https://github.com/rtCamp/nginx-helper/issues/307

## Testing
The code has been tested locally and is working as expected.

### CODE REVIEW

File: admin/class-nginx-helper-admin.php [hunk 1]
```diff
@@ -281,6 +281,28 @@ public function nginx_helper_default_settings() {
 		);
 
 	}
+    
+    public function store_default_options() {
+        $options = get_site_option( 'rt_wp_nginx_helper_options', array() );
+        $default_settings = $this->nginx_helper_default_settings();
+        
+        $removable_default_settings = array(
```
Comment (SH4LIN) [inline admin/class-nginx-helper-admin.php:289]: Why are we removing these settings from the list of settings to be stored in the options table?

**Review by SH4LIN — CHANGES_REQUESTED** [review]

File: admin/class-nginx-helper-admin.php [hunk 2]
```diff
@@ -281,6 +281,28 @@ public function nginx_helper_default_settings() {
...
+            'redis_database',
+            'redis_unix_socket'
+        );
+        
+        // Remove all the keys that are not to be stored by default.
+        foreach ( $removable_default_settings as $removable_key ) {
+            unset( $default_settings[ $removable_key ] );
+        }
+        
+        $diffed_options = wp_parse_args( $default_settings, $options );
```
Comment (SH4LIN) [inline admin/class-nginx-helper-admin.php:302]: According to the documentation and function signature, keep the default options as the second argument and the options (`$args`) as the first.

**Review by SH4LIN — APPROVED** [review]
LGTM!

**Comment by Vedant-Gandhi** [comment]
@SH4LIN The action is throwing an error because it is utilising a deprecated version of the upload artifact API.

**Pull request merged**